sdk/servicebus/azure-messaging-servicebus/swagger/servicebus-swagger.json

```diff
@@ -767,6 +767,15 @@
             "name": "ForwardDeadLetteredMessagesTo",
             "namespace": "http://schemas.microsoft.com/netservices/2010/10/servicebus/connect"
           }
+        },
+        "MaxMessageSizeInKilobytes": {
```

Comment:
Thanks for making this change as well. We need to make sure that swagger is always in line with the SDK changes

Reply:
Yeah, I run autorest.java to generate sdk by using these swagger, everything is ok.